server/graphql.ts

```diff
@@ -242,6 +242,14 @@ function resolver(registration: Registration): IResolver {
                     return null;
                 }
 
+                // If warnOnDuplicates is not set for this tag, set it to true (most restrictive option)
+                if (!tagDetails.hasOwnProperty("warnOnDuplicates")
```

Comment:
Correct me if I'm wrong but can't this be simplified to `!tagDetails.warnOnDuplicates` which will evaluate to `true` if the property is `null`, `false`, or `undefined` because it hasn't been set.